src/modules/modals/components/TokenModal.tsx

```diff
@@ -121,6 +121,10 @@ const TokenModal: React.FC<Props> = ({ onSubmit, onCancel, type, token, maxAmoun
                 return <FormattedMessage id="modalDepositTitle" defaultMessage="Deposit {symbol}" values={{ symbol: token.symbol.code }} />
             case `WITHDRAW`:
```

Comment:
@MrToph I was checking the code looks ok, just one comment, do you think maybe we can have a constant file for set WITHDRAW, BORROW, PAYOFFDEBT, DEPOSIT, string values, to be used thought the app, instead set hardcoded this values?

Reply:
I don't think it's necessary when using TypeScript. Actually, we kind of have this here by using the [TS enum](https://github.com/vigorstablecoin/vigor-app/pull/17/files#diff-ea7490564896a922af8438ab94337ac6R2). It will warn you when you use a string that is not included in the enums.